src/models/thelio-mira-b1.0/repairs.md

```diff
@@ -151,11 +151,11 @@ Thelio Mira supports up to two dual-slot GPUs. All PCIe x16 slots run at Gen 4 s
 
 ![GPU brace screws](./img/gpu-brace-plastic-screws.webp)
 
-_The plastic GPU braces' primary function is to prevent damage during shipping. The system can be run without the braces if they don't fit an upgraded card; the back PCIe bracket provides primary support for the GPUs._
+_The GPU brace assembly's primary function is to prevent damage during shipping. The system can be run without the brace if it doesn't fit an upgraded card; the back PCIe bracket provides primary support for the GPUs._
```

Comment:
This line's revision is more than plural -> singular, this is changing from talking about the plastic pieces to talking about the entire metal brace assembly. Would we not still recommend putting the metal brace in to support the CPU shroud even if the plastic pieces are removed to make way for a larger GPU?

Reply:
I don't think the CPU shroud needs additional support in situations where the GPU doesn't need additional support. It's a lot more sturdy and a lot less delicate. We can change it back if you think it's better to keep it installed though.

Reply:
@leviport I've revised this section. I suppose people are free to recognize for themselves that they can place a partial GPU brace back into their system if they want to. The more important point of the note is to assure customers that not having a fitting GPU brace is not an issue for normal operation of the system.

Also added a highlight for the depth adjustment screws/screwholes. (I thought about whether the rest of the screwholes for the height adjustment should also be highlighted to match, but I think it looks clear as-is.)

Reply:
Fixed the typo you pointed out, thank you.